server/server/Models/restaurant.js

```diff
@@ -0,0 +1,41 @@
+const mongoose = require("mongoose");
+
+// schema
+const restaurantSchema = new mongoose.Schema({
+  _id: mongoose.Schema.Types.ObjectId,
+
+  name: {
+    type: String,
+    required: true,
+  },
+
+  location: {
+    type: [String],
+    required: true,
+  },
+
+  phoneNumber: {
+    type: String,
+    required: true,
+  },
+
+  restaurantPicture: {
+    // type to be decieded later on
+    // type: String,
+    required: true,
```

Comment:
we can put a placeholder image in frontend if restaurant image is absent from database